### ISSUE TITLE
[Bug]: Variable display error

### ISSUE THREAD

**Issue by Kathieux** [jasp-stats/jasp-issues#2703]
### JASP Version

0.18.3

### JASP Module

Unrelated

### What analysis are you seeing the problem on?

General display of variables for all analyses

### What OS are you seeing the problem on?

Windows 10

### Bug Description

Hi, 
I am opening a .csv file which I transformed from .xlsx. All data is shown normally, but when I want to do analyses, the variables are not displayed correctly (see attached file).
I tried different .csv formats, installed JASP anew, restarted computer, but nothing changed the issue - what am I missing? 
I would be very happy about suggestions/comments!

![image](https://github.com/jasp-stats/jasp-issues/assets/127356936/7bec0057-a4dc-450d-aa28-8501f44115f4)

### Expected Behaviour

Normal display of variables.

### Steps to Reproduce

1. Just opening the file.
2.
3.
...

### Final Checklist

- [X] I have included a screenshot showcasing the issue, if possible.
- [X] I have included a JASP file (zipped) or data file that causes the crash/bug, if applicable.
- [X] I have accurately described the bug, and steps to reproduce it.

**Comment by tomtomme** [comment]
Can you share the CSV file please? That would help us diagnosing the problem

**Comment by Kathieux** [comment]
Hi, 
sorry, yes here it comes:
[dataset_sample.csv](https://github.com/jasp-stats/jasp-issues/files/15218789/dataset_sample.csv)

**Comment by tomtomme** [comment]
There are names in the file. Are you sure that this is okay?

**Comment by Kathieux** [comment]
Yes that's ok, those are public figures from wikipedia articles. But I can also delete them I don't think they matter for the problem.

**Comment by tomtomme** [comment]
No, then its fine. 
So I can confirm the bug in 0.18.3 and 0.19beta (flatpak). 
I have no idea where this comes from. 
I changed scales, resorted the var list, renamed to get rid of ( or . and shortened names. Nothing helps. I have never seen this before.
@boutinb : Any idea?

**Comment by shun2wang** [comment]
Hi @Kathieux thanks for the report!

There are many spaces and newlines at the end of the variable names in your example file. This is the main reason. You can temporarily fix this problem by simply editing to remove these spaces and newlines. a fix will be including in the next release.

**Comment by Kathieux** [comment]
Hi @shun2wang, 

thanks so much, that solved the issue! I wonder how they got there, but great to have a solution!! :)
And thanks to @tomtomme for your support!

**Comment by shun2wang** [comment]
>I wonder how they got there

Well, it's hard to say but maybe some "newlines" in your original excel files(usually generated when copy-paste from website) or they were added while exporting.
